subm/src/subm.js

```diff
@@ -427,23 +521,48 @@ async function main(env, { clock, get, express, makeStorage, admin }) {
       error: app.get('env') === 'development' ? err : {},
     });
   };
-  // Upload form
-  // Note the actual upload request goes directly to Google Cloud Storage.
-  app.get(Site.path.uploadSlog, loginCheck, async (req, res) => {
-    try {
-      const participant =
-        /** @type { ReturnType<typeof makeTestnetParticipant> } */ (req.user);
-      const policy = await participant.uploadPolicy(
-        new Date(clock()),
-        Site.uploadSuccessURL(base, req.protocol, req.hostname),
-      );
-      const files = await participant.myFiles();
-      const page = Site.uploadSlog(participant.member, policy, files);
-      res.send(page);
-    } catch (err) {
-      handleError(res, req.baseUrl, err);
+
+  app.get(Site.path.contactForm, loginCheck, async (req, res) => {
+    const contact = /** @type { ReturnType<typeof makeContact> } */ (req.user);
+    const { user } = contact.member;
+    if (!user) throw Error('no user');
+    const detail = await discordAPI.users(user.id);
+    let email = detail.email; // ISSUE: why is email missing? what bot permission am I missing?
+    if (!email) {
+      email = await contact.getEmail();
```

Comment:
We should probably always fetch this anyway, in case they already updated their email, we shouldn't prepopulate with the Discord email.